### PR TITLE
Expose Keras-Style API Guide

### PR DESCRIPTION
**Pull request by Deliadd**
update Readme and Activation.

### CODE REVIEW

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+model.add(Activation('relu'))
+```
+```python
+model.add(Dense(32, activation="relu"))
+```
+
+
+---
+## **Available Activations**
+* [relu](../../APIGuide/Layers/Activations/#relu)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: The link is invalid

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* [tanh](../../APIGuide/Layers/Activations/#tanh)
+* [sigmoid](../../APIGuide/Layers/Activations/#sigmoid)
+* [hard_sigmoid](../../APIGuide/Layers/Activations/#hardsigmoid)
+* [softmax](../../APIGuide/Layers/Activations/#softmax)
+* [softplus](../../APIGuide/Layers/Activations/#softplus)
+* [softsign](../../APIGuide/Layers/Activations/#softsign)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: Seems there are more activations now (such as relu6)

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+## **Activation**
```
Comment (zhichao-li) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: Let's empty this file. Those file should be treated as a placeholder in this PR only.

**Comment by zhichao-li** [comment]
I was thinking the `KerasStyleAPI` hasn't been merged or revert by others, but seems like the content has already been there, so you only need to simply expose this link on the website and add whatever you need in a follow up PR.

**Comment by hkvision** [comment]
There are files already there. Just enable the links in `mkdocs.yml` and add/update a layer doc for others to refer to.

**Comment by zhichao-li** [comment]
It turn out that #586 is good, but somehow #603 deleted the yml config. Let's add it back......
